src/object/class_object.c

```diff
@@ -4045,7 +4064,75 @@ classobj_find_cons_index2_col_type_list (SM_CLASS_CONSTRAINT * cons, OID * root_
   return key_type;
 }
 
+#if defined(SUPPORT_DEDUPLICATE_KEY_MODE)
+static void
+classobj_check_attr_in_unique_constraint (SM_CLASS_CONSTRAINT * cons_list, DB_CONSTRAINT_TYPE new_cons,
+					  char **att_names, int *asc_desc, SM_FUNCTION_INFO * func_index_info)
+{
+  SM_CLASS_CONSTRAINT *cons;
+  SM_ATTRIBUTE **attp;
+  char **namep;
+  int nnames, cols_non_func;
+
+  // If there is a column corresponding to PK among the attributes constituting the index, the reserved_index_column is not added.
+  int deduplicate_key_col_pos = -1;
+  for (nnames = 0, namep = att_names; *namep; namep++, nnames++)
+    {
+      if (IS_DEDUPLICATE_KEY_ATTR_NAME (*namep))
+	{
+	  deduplicate_key_col_pos = nnames;
+	}
+    }
+
+  if (deduplicate_key_col_pos == -1)
+    {
+      return;
+    }
 
+  cols_non_func = (func_index_info) ? func_index_info->attr_index_start : nnames;
+
+  for (cons = cons_list; cons; cons = cons->next)
+    {
+      if (SM_IS_CONSTRAINT_UNIQUE_FAMILY (cons->type) == false)
+	{
+	  continue;
+	}
+
+      if (!cons->attributes)
+	{
+	  continue;
+	}
+
+      for (attp = cons->attributes; *attp; attp++)
+	{
+	  int idx;
+	  bool found = false;
+	  for (idx = 0, namep = att_names; *namep && idx < cols_non_func; namep++, idx++)
+	    {
+	      if (intl_identifier_casecmp ((*attp)->header.name, *namep) == 0)
+		{
+		  found = true;
+		  break;
+		}
+	    }
+
+	  if (found == false)
```

Comment:
it can be removed from above suggestion.

Reply:
It was explained verbally.
The code remains as is.